### DISCUSSION TITLE
The inconsistent feature dimension size of WikipediaNetwork dataset

### DISCUSSION THREAD

**Discussion by ZhuYun97** [pyg-team/pytorch_geometric#2981] [Q&A]
# The inconsistent feature dimension size of Chameleon and Squirrel dataset

The feature dimension size of Chameleon and Squirrel dataset is 2325 and 2089 in pyg 1.7.0. But in the latest pyg version(1.7.2), the feature size both are 128 for Chameleon and Squirrel dataset which is not the same as reported in [ref1](https://arxiv.org/abs/2002.05287).
![image](https://user-images.githubusercontent.com/37068560/129560098-00def02d-00f0-498e-acaa-d85efe5cc1ff.png)
[ref1](https://arxiv.org/abs/2002.05287)

## Environment

* PyTorch Geometric version: 1.7.2

## Additional context
If I want to use the original dimension size(2325 and 2089), is it the only way to degrade pyg version?

**Comment by rusty1s** [comment]
See https://github.com/rusty1s/pytorch_geometric/issues/2979

**Reply by ZhuYun97** [reply]
Thanks, but no one has answered it.
